File: Task2/src/HospitalityHub.Core/DTOs/Booking/FilterBookingResponse.cs
using HospitalityHub.Core.Enums;

namespace HospitalityHub.Core.DTOs.Booking;

public class FilterBookingResponse
{
    public int BookingId { get; set; }
    public int CustomerId { get; set; }
    public int RoomId { get; set; }
    public DateTime CheckIn { get; set; }
    public DateTime CheckOut { get; set; }
    public decimal TotalPrice { get; set; }
    public decimal TotalDiscountPercent { get; set; }
    public bool IsPaid { get; set; }
    public ERoomType RoomType { get; set; }
    public int NumberOfAdults { get; set; }
    public int NumberOfChildren { get; set; }
}

File: apzkr-pzpi-21-3-merkulov-arsenii/Task3-WebClient/hospitality-hub/src/app/api-proxy/models/filter-booking-response.ts
/* tslint:disable */
/* eslint-disable */
import { ERoomType } from '../models/e-room-type';
export interface FilterBookingResponse {
  bookingId?: number;
  checkIn?: string;
  checkOut?: string;
  customerId?: number;
  isPaid?: boolean;
  numberOfAdults?: number;
  numberOfChildren?: number;
  roomId?: number;
  roomType?: ERoomType;
  totalDiscountPercent?: number;
  totalPrice?: number;
}
